### PR TITLE
feat: add `circl@1.3.8`

### PR DESCRIPTION
**Pull request by mattyclarkson**

### CODE REVIEW

**Comment by bazel-io** [comment]
Hello @bazelbuild/bcr-maintainers, modules without existing maintainers (circl) have been updated in this PR. Please review the changes.

File: modules/circl/1.3.8/MODULE.bazel [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+bazel_dep(name = "gazelle", version = "0.36.0")
+bazel_dep(name = "rules_cc", version = "0.0.9")
+bazel_dep(name = "rules_go", version = "0.47.2")
+
+# TODO: remove this when `purego` support is actually released in `rules_go`
+git_override(
+    module_name = "rules_go",
+    remote = "https://github.com/bazelbuild/rules_go.git",
+    commit = "fcd6390936ac61d76b5c00f1f33a505045d11888",
+)
```
Comment (mattyclarkson) [inline modules/circl/1.3.8/MODULE.bazel:15]: I will remove this, the `0007` patch when we get a `rules_go` release.

**Comment by leungster** [comment]
Any updates on this? I'd like to update my circl version without needing to patch it.

**Comment by bazel-io** [comment]
Hello @bazelbuild/bcr-maintainers, modules without existing maintainers (circl) have been updated in this PR. Please review the changes.

File: modules/circl/1.3.8/MODULE.bazel (hunk 1 above)
Comment (mattyclarkson) [reply modules/circl/1.3.8/MODULE.bazel]: `0.48.0` is released and works. Override patch has been dropped.

**Comment by mattyclarkson** [comment]
Not sure what is going on in the CI and why it is still picking up `rules_go@0.47.2`. I'll have another look soon.

File: modules/circl/1.3.8/source.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+    "url": "https://proxy.golang.org/github.com/cloudflare/circl/@v/v1.3.8.zip",
+    "integrity": "sha256-gCOgRORw9SLiykG62YsYrW0TTWiBABsRIK0ukWbNpuY=",
+    "patches": {
+        "0001-build-add-bzlmod-files.patch": "sha256-VJjrWhKx38f80vGBjwMgXQHiTaidlX1l59F8T/pi5dE=",
```
Comment (fmeum) [inline modules/circl/1.3.8/source.json:5]: You can reuse old patches via `../../<other_version>/patches/patch`, which may help reduce the diff.

**Review by fmeum — APPROVED** [review]

**Comment by albertocavalcante** [comment]
Linking to this PR: https://github.com/cloudflare/circl/issues/537#issuecomment-2738664357

**Pull request merged**